.prettierrc

```diff
@@ -1,5 +1,4 @@
 {
-  "arrowParens": "avoid",
```

Comment:
Better mark this sneak-in prettier format setting in the PR description.

Reply:
oh yes. only included as a single commit but not in PR description.